hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -788,7 +788,7 @@
       // Edn connections
       'edn0.edn'              : ['keymgr.edn', 'otp_ctrl.edn', 'ast_edn.edn', 'kmac.entropy',
                                  'alert_handler.edn', 'aes.edn'],
-      'edn1.edn'              : ['otbn.edn'],
+      'edn1.edn'              : ['otbn.edn_rnd', 'otbn.edn_urnd'],
```

Comment:
We need the URND one on edn0 (feeding the local LFSR), and the RND one on edn1.

Reply:
Well this is what I was unsure about as I said in the initial comment.

@tjaychen could you clarify? As you'd just upped the total connections from 4 to 6 I was't sure if there was an oversight of not moving to 7 so we'd have room for an OTBN connection or if they're both intended to go on edn1 for now.

Reply:
I'm pretty sure they need to go to edn0 and edn1, respectively. Tim only rearranged existing ones, and didn't add the second OTBN one. The picture in https://github.com/lowRISC/opentitan/issues/5487 shows how it should look.

Reply:
Yeah they need to go to different EDN's eventually. But it could be that @tjaychen is planning to rework these connections anyways, so it may be fine to just connect both to EDN1 now...

Reply:
The reordering was done yesterday in https://github.com/lowRISC/opentitan/pull/5505, the current setup should match what we're looking for, modulo the second OTBN one.

Reply:
Oh I see. Yeah in that case they should go to different EDNs.

Reply:
Updated to link urnd to edn0 and rnd to edn1